pyproject.toml

```diff
@@ -1,28 +1,81 @@
-[tool.poetry]
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+
+[project]
 name = "docker-snapshot"
-homepage = "https://github.com/occamz/ds"
-documentation = "https://github.com/occamz/ds"
-version = "1.0.11"
+dynamic = ["version"]
 description = "`ds` is a development utility for managing snapshots inside a docker container."
-authors = ["occamz"]
 readme = "README.md"
+requires-python = ">=3.9"
+license = "Apache-2.0"
+keywords = []
+authors = [{ name = "occamz" }]
+classifiers = [
+    "Development Status :: 4 - Beta",
```

Comment:
Also might want to comment on this (e.g. see [the full list](https://pypi.org/classifiers) if you want this to be something else).

Reply:
Same as above, I think it is fine.